Specs/Scene/LiteralColorExpressionSpec.js

```diff
@@ -0,0 +1,64 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/LiteralColorExpression',
+        'Core/Color'
+    ], function(
+        LiteralColorExpression,
+        Color) {
+    'use strict';
+
+    function MockStyleEngine() {
+    }
+
+    MockStyleEngine.prototype.makeDirty = function() {
+    };
+
+    it('constructs with hex', function() {
+        var expression = new LiteralColorExpression(new MockStyleEngine(), "#ffffff");
```

Comment:
Use single quotes for all the strings in this file.  We usually always use single except for JSON examples, e.g., the CZML examples in Sandcastle, and the style examples we have been making in Sandcastle.